src/sentry/replays/query.py

```diff
@@ -320,21 +319,18 @@ def _grouped_unique_values(
     )
 
 
-def take_first_from_aggregation(
+def take_any_from_aggregation(
     column_name: str,
     alias: Optional[str] = None,
     aliased: bool = True,
 ) -> Function:
-    """Returns the first value encountered in an aggregated array.
-
-    E.g.
-        [1, 2, 2, 3, 3, 3, null] => [1] => 1
+    """Returns any value of a non group-by field. in our case, they are always the same,
+    so the value should be consistent.
```

Comment:
If you really want to take the first from the aggregation, you can use something like `argMin(sequence_id, column_name)` since it's sorted by the `sequence_id` but sounds like `any` should be fine as well for your use case